index.js

```diff
@@ -41,6 +40,21 @@ module.exports = (gulp, userConfig, tasks) => {
   // We basically need icons compiled before CSS & CSS/JS compiled before inject:pl before pl
   // compile. The order of the `require`s above is the order that compiles run in; not perfect, but
   // it works.
+
+  // Users can potentially disable all tasks. This will leave empty task arrays
+  // which causes Gulp to kersplode. Add at least one dummy task to the array if
+  // it is otherwise empty
+  // The major types are: compile, watch, validate, clean, default
+  _.forEach(tasks, (value, key) => {
+    if (tasks[key].length === 0) {
+      // Create a fake task that just signals completion via callback
+      gulp.task(`${key} fallback task`, (cb) => { cb(); });
+      // Add it on if otherwise empty
+      tasks[key].push(`${key} fallback task`);
```

Comment:
You should probably make the output a bit more explicit because this might be inferred by developers as something that happens *in addition* to the normal behavior, or as the result of an obscure bug.

For example: "${key}: No action was taken per project configuration."